static/sass/partials/_header.scss

```diff
@@ -25,6 +135,14 @@ header {
 
 @media (min-width: 768px) {
```

Comment:
Please add a small comment to describe what kind of device this breakpoint is targeting.